feg/cloud/go/services/feg_relay/gw_to_feg_relay/servicers/session_proxy_relay.go

```diff
@@ -129,6 +129,26 @@ func (s *RelayRouter) getSessionProxyClient(
 	return protos.NewCentralSessionControllerClient(conn), ctx, cancel, nil
 }
 
+//N7_proxy service for 5G
```

Comment:
i think you do not need different functions for each client. They are in fact only check if they are error, for you to check again on the main function

```
func (s *RelayRouter) getProxyClientctx context.Context, r protos.RATType, imsi string) (protos.CentralSessionControllerClient, context.Context, context.CancelFunc, error) {
	conn, ctx, cancel, err := s.GetFegServiceConnection(c, imsi, getProxyType(r))
	if err != nil {
		return nil, nil, nil, err
	}
	return protos.NewCentralSessionControllerClient(conn), ctx, cancel, nil

func getProxyType(r protos.RATType){
	switch(r):{
	case protos.RATType_TGPP_NR:
		return FegN7Proxy
	default:
		return FegSessionProxy
		
	}
}



```